hub/src/server/drivers/AzDriver.js

```diff
@@ -69,7 +69,7 @@ class AzDriver implements DriverModel {
     return `https://${this.accountName}.blob.core.windows.net/${this.bucket}/`
   }
 
-  listBlobs(prefix: string, page: ?string) {
+  listBlobs(prefix: string, page: ?string) : Promise<{ entries: Array<string>, page: any }> {
```

Comment:
Can we create a `type` for `{ entries: Array<string>, page: any }` and use that here, instead of repeating this structure in all the drivers?  You can add it to the driver model.

Reply:
Good idea 👍